src/api.rs

```diff
@@ -2977,4 +2984,51 @@ mod test {
       }
     );
   }
+
+  #[interpolate_test(1, 1)]
+  #[interpolate_test(2, 2)]
+  #[interpolate_test(3, 3)]
+  fn output_frameno_no_scene_change_at_flashes(flash_at: u64) {
```

Comment:
Could you also add a test for the second behavior (AAAAABBCCDDEEFFFFFF)? And it would be super nice to have these tests additionally check the boundary conditions for the flash detection (so the flash that spans the maximum number of frames for it to still be considered a flash, and then a flash that spans one more frame so it is considered a legit scene cut).